content/blog/2020/06/2020-06-14-repellent-terminology.adoc

```diff
@@ -0,0 +1,32 @@
+---
+layout: post
+title: "Repellent Terminology"
+tags:
+- community
+- opensource
+author: markyjackson-taulia
+description: "Terminology Updates"
+opengraph:
+  image: /images/docker/dockerJenkins_social.png
+---
+
+We have to talk, a real conversation about the power of words, honest talk about hurtful words, and the energy they hold.
+
+As a community, we cannot turn a blind eye to what is happening in the world. 
+What transpired on May 25th, 2020, and witnessed around the world has forever changed me as a person. 
+George Floyd was a human and did not deserve what happened. Many before him did not deserve what happened.
+
+Injustice must be addressed in every action. It must be at the forefront of our minds. We must do better.
+
+In 2016, the Jenkins community decided to start removing offensive terminology within the project. Offensive language was deprecated in Jenkins 2.0.
+In 2017, the project began tracking areas for correction link:https://issues.jenkins-ci.org/browse/JENKINS-42816[tracking areas for correction].
+This year, a group of core contributors continued addressing this critical link:https://www.jenkins.io/blog/2020/05/06/docker-agent-image-renaming/[work]. 
+
+The Advocacy & Outreach SIG met to discussed and prioritize the continued work. 
+This week we took another step towards removing offensive terminology within the project by updating previous blog posts and link:https://github.com/jenkins-infra/jenkins.io/pull/3447[removing offensive terminology].
+There is more link:https://groups.google.com/forum/#!topic/jenkinsci-dev/CLR55wMZwZ8[work to do].
+
+I am sorry these hurtful words exist in our community. I am sorry for the hurt they have caused.
```

Comment:
I'd add a call to action at the end of this phrase.

`Let's do better, together.`
(or something like that)